pkg/containerd/cmd/service/system_init.go

```diff
@@ -71,6 +76,32 @@ func systemInit(args []string) {
 		os.Exit(1)
 	}
 
+	// remove (unlikely) old containerd socket
+	_ = os.Remove(*sock)
+
+	// start up containerd
+	cmd := exec.Command(*binary)
+	cmd.Stderr = os.Stderr
```

Comment:
I don't believe containerd outputs anything to stdout - I was originally sending stdout but it was silent, and I discovered that it is only logging to stderr (which seems correct), and so I set it up like this.